engines/python/setup/djl_python/deepspeed.py

```diff
@@ -95,11 +95,11 @@
 
 
 def get_torch_dtype_from_str(dtype: str):
-    if dtype == "fp32":
+    if dtype == "float32":
```

Comment:
I'm not sure if full name is better.
seems fp32, fp16, bf16 are well accepted name already